.tekton/pipeline/acceptance-tests.yaml

```diff
@@ -107,6 +109,35 @@ spec:
           workspace: shared-workspace
         - name: source
           workspace: source
+    - name: tekton-results-fetch-sha
+      taskRef:
+        name: tekton-results-fetch-sha
+      runAfter:
+        - "clone-pipeline-service-git"
+      params:
+        - name: target_branch
+          value: $(params.target_branch)
+      workspaces:
+        - name: source
+          workspace: source
+    - name: tekton-results-upstream-e2e
+      taskRef:
+        name: tekton-results-upstream-e2e
+      runAfter:
+        - "tekton-results-fetch-sha"
+        - "plnsvc-setup"
+      params:
+        - name: target_branch
+          value: $(params.target_branch)
+        - name: repo_url
+          value: "$(params.results_url)"
+        - name: revision
+          value: "$(tasks.tekton-results-fetch-sha.results.tekton-results-downstream-sha)"
+      workspaces:
+        - name: source
+          workspace: tekton-results-source
```

Comment:
@avinal The pipeline-service-test failure is because of workspace
```code
error: the path "/workspace/source/test/e2e/kustomize/rbac.yaml" does not exist
error: failed to create token: serviceaccounts "all-namespaces-read-access" not found
error: failed to create token: serviceaccounts "single-namespace-read-access" not found
```

you have not cloned the `tekton-results` repo into `tekton-results-source` workspace. So its not able to find the expected files.

Reply:
Let me fix that, thanks